### PR TITLE
completed Products Page

### PR DESCRIPTION
**Pull request by GarvG**
Please Review My PR!

### CODE REVIEW

File: Products.html [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+                        <img src="IMAGES/whislist.jpg" alt="">
+                        <span class="material-icons heart">
+                            favorite_border
+                            </span>
+                    </div>
+                   
+                    <h4>Jordan-High</h4>
+                    <h4>Rs 19,999</h4>
+                    <button class="MTCart">MOVE TO CART</button>
+                </div> -->
```
Comment (zoef-yt) [inline Products.html]: remove commented code.

File: products.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    display: flex;
+}
+.sidebar
+{
+    margin-left: 4rem;
+    width: 16rem;
+    box-shadow: 0px 0px 11px 0px var(--boxshawdow);
+    padding: 10px;
+}
+
```
Comment (zoef-yt) [inline products.css]: remove space between line 12 to 14

**Review by zoef-yt — CHANGES_REQUESTED** [review]
Please write a better description for PR. Add images or gif.

File: products.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    display: flex;
+    justify-content: space-between;
+    width: 14rem;
+}
+input[type='range']
+{
+    background-color: var(--black);
+    width: 14rem;
+}
+h2{
```
Comment (Muzammil-khan710) [inline products.css]: Change line of h2 like you have given spaces to previous elements

File: products.css [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+.image
+{
+    position: relative;
+}
+.heart
+{
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    color:var(--removeC);
```
Comment (Muzammil-khan710) [inline products.css:95]: Remove space

**Review by Muzammil-khan710 — APPROVED** [review]
Left some comments do check !.. and after that it is good to merge.. Approving as of now

**Review by purvasharmadev — APPROVED** [review]
Code is looking good, do the suggested changes and you are good to go.

File: Products.html (hunk 1 above)
Comment (GarvG) [reply Products.html]: done

File: products.css (hunk 2 above)
Comment (GarvG) [reply products.css]: Removed

File: products.css (hunk 4 above)
Comment (GarvG) [reply products.css:95]: Done

File: products.css (hunk 3 above)
Comment (GarvG) [reply products.css]: Done

**Review by zoef-yt — APPROVED** [review]

**Pull request merged**